agent/php_newrelic.c

```diff
@@ -120,6 +120,10 @@ ZEND_BEGIN_ARG_INFO_EX(newrelic_arginfo_void, 0, 0, 0)
 ZEND_END_ARG_INFO()
 #endif /* PHP 8.0+ */
 
+ZEND_BEGIN_ARG_INFO_EX(newrelic_get_request_metadata_arginfo, 0, 0, 0) 
```

Comment:
Should this be 0 or 1:
```suggestion
ZEND_BEGIN_ARG_INFO_EX(newrelic_get_request_metadata_arginfo, 0, 0, 1) 
```

Reply:
I guess 0 is fine because it states how many required arguments a function takes. Since all arguments to `newrelic_get_request_metadata` are optional, 0 is fine.

Reply:
Correct :)